common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -2838,13 +2809,30 @@ func workflowExecutionFromRow(result map[string]interface{}) (*p.InternalWorkflo
 	if protoInfo.ReplicationData != nil {
 		protoReplVersions, err := ProtoReplicationVersionsFromResultMap(result)
 		if err != nil {
-			return nil, nil, err
+			return nil, err
 		}
 
 		state = ReplicationStateFromProtos(protoInfo, protoReplVersions)
 	}
 
-	return info, state, nil
+	// var vh *history.VersionHistories
```

Comment:
Why these are commented out?

Reply:
In case these tests fail :) I removed fetching it from the column and used the proto object that is stored already in this change.